app/scripts/controllers/metametrics.test.ts

```diff
@@ -512,10 +558,14 @@ describe('MetaMetricsController', function () {
       const metaMetricsController = getMetaMetricsController();
 
       await expect(
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore because we are testing the error case
```

Comment:
If you use `ts-expect-error` then you don't need the `ban-ts-comment` override:

``` suggestion
        // @ts-expect-error because we are testing the error case
```

(The reason why `ts-ignore` is banned is because it outright ignores the type error, whereas `ts-expect-error` checks whether one exists and gives you an error if there is in fact no error)